sdk/samples/storage/blobs/CMakeLists.txt

```diff
@@ -15,15 +15,17 @@ if(TRANSPORT_CURL)
   endif()
 endif()
 
-# Generate map file
-if(MSVC)
-    add_link_options(/MAP)
-elseif(CMAKE_C_COMPILER_ID MATCHES "Clang")
-    add_link_options(-Wl,-map,blobs_client_example.map)
-    add_link_options(-Os)
-else()
-    add_link_options(-Xlinker -Map=blobs_client_example.map)
-    add_link_options(-Os)
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13")
```

Comment:
1) Move to .cmake module file to avoid duplication.
2) Is there no way to make this map file generation work on 3.10?

Reply:
@vhvb1989, @barcharcraz do you know of a way to set the cmake flags directly?